RobotCode/snobot2018/simulator_config/simulator_config.yml

```diff
@@ -38,7 +38,7 @@ mPwm:
     mGearReduction: 12.0
     mGearboxEfficiency: 1.0
     mHasBrake: false
-    mInverted: false
+    mInverted: true
```

Comment:
what is this?

Reply:
That is for the simulator. Due to how the drivetrain works one wheel is "backwards"